### PR TITLE
Adding the submission info to the session

### PR DESCRIPTION
**Pull request by Conorc1000**
relates to #4705 on the main app

### CODE REVIEW

File: src/TableView.elm [hunk 1]
```diff
@@ -368,6 +381,73 @@ viewCells dateWithSessions model timeDelimiters numColumns timeDelimiter =
...
+            toString submission.id
+
+        filterFunc submissionId submission =
+            submissionId == submission.id
+
+        submissionInfo =
+            List.filter (filterFunc submission.id) model.submissions
+                |> List.head
+
+        submissionTitle =
```
Comment (roryc89) [inline src/TableView.elm]: here to line 441 can be:
```elm
(submissionTitle, programmeCode) =
  submissionInfo |> Maybe.map \s -> (s.title, s.programmeCode) |> Maybe.withDefault ("","")
```

File: src/MainUpdate.elm [hunk 2]
```diff
@@ -456,7 +456,11 @@ update msg model =
                     updatedModel ! [ Task.perform (UpdateShowMobileView) Window.size ]
 
             UpdateModel (Err str) ->
-                model ! [ Task.perform (UpdateShowMobileView) Window.size ]
+                let
+                  debug = Debug.log "Err" (Err str)
```
Comment (roryc89) [inline src/MainUpdate.elm]: Not a biggie but we shouldn't really being including logs in production

File: src/TableView.elm [hunk 3]
```diff
@@ -368,6 +381,73 @@ viewCells dateWithSessions model timeDelimiters numColumns timeDelimiter =
         |> List.indexedMap (viewCell dateWithSessions model timeDelimiters numColumns timeDelimiter)
 
 
+add0ToSingleDigits number =
```
Comment (roryc89) [inline src/TableView.elm]: I think this function may already exist

Comment (Conorc1000) [reply src/TableView.elm]: in the main application with have a v.similar function

**Pull request merged**